<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>简聊</string>
  </property>
  <widget class="QPushButton" name="loginPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>311</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>登 录</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="usrLineEdit">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>274</y>
     <width>263</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>350</y>
     <width>263</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="selectLoginButton">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>8</y>
     <width>218</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="selectSignButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>8</y>
     <width>218</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bgLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>550</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>311</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>注 册</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="s_usrLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>182</y>
     <width>263</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="s_EmailLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>263</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="s_phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>298</y>
     <width>263</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="s_pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>357</y>
     <width>263</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="s_noiticeLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注：联系方式设置后不可修改、不可重复</string>
   </property>
  </widget>
  <zorder>bgLabel</zorder>
  <zorder>loginPushButton</zorder>
  <zorder>usrLineEdit</zorder>
  <zorder>pwdLineEdit</zorder>
  <zorder>selectLoginButton</zorder>
  <zorder>selectSignButton</zorder>
  <zorder>signPushButton</zorder>
  <zorder>s_usrLineEdit</zorder>
  <zorder>s_EmailLineEdit</zorder>
  <zorder>s_phoneLineEdit</zorder>
  <zorder>s_pwdLineEdit</zorder>
  <zorder>s_noiticeLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
